src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1307,26 +1386,31 @@ public static void updateKitError(@NonNull long kitRequestId, @NonNull String me
         });
 
         if (results.resultException != null) {
-            throw new RuntimeException("Error updating error/message of kitRequest w/ dsm_kit_request_id " + kitRequestId, results.resultException);
+            throw new RuntimeException("Error updating error/message of kitRequest w/ dsm_kit_request_id " + dsmKitRequestId, results.resultException);
         }
+
+        KitRequestShipping kitRequestShipping = new KitRequestShipping(dsmKitRequestId, null, null, null, null, message);
+
+        UpsertPainlessFacade.of(DBConstants.DDP_KIT_REQUEST_ALIAS, kitRequestShipping, ddpInstanceDto, "dsmKitRequestId", "dsmKitRequestId", dsmKitRequestId)
```

Comment:
can we use ESObjectConstants.DSM_KIT_REQUEST_ID for `dsmKitRequestId` here as well